types/react-native/index.d.ts

```diff
@@ -9616,15 +9616,57 @@ export const ShadowPropTypesIOS: ShadowPropTypesIOSStatic;
 
 export const DeviceEventEmitter: DeviceEventEmitterStatic;
 
+/**
+ * The React Native implementation of the IOS RCTEventEmitter which is required when creating
+ * a module that communicates with IOS
+ */
+type NativeModule = {
+    /**
+     * Add the provided eventType as an active listener
+     * @param eventType name of the event for which we are registering listener
+     */
+    addListener: (eventType: string) => void;
+
+    /**
+     * Remove a specified number of events.  There are no eventTypes in this case, as 
+     * the native side doesn't remove the name, but only manages a counter of total
+     * listeners
+     * @param count number of listeners to remove (of any type)
+     */
+    removeListeners: (count: number) => void;
+};
+
 /**
  * Abstract base class for implementing event-emitting modules. This implements
  * a subset of the standard EventEmitter node module API.
  */
-export class NativeEventEmitter extends EventEmitter {
+declare class NativeEventEmitter extends EventEmitter {
+    /**
+     * @param nativeModule the NativeModule implementation.  This is required on IOS and will throw
+     *      an invariant error if undefined.
+     */
+    constructor(nativeModule?: NativeModule | undefined)
```

Comment:
`constructor(nativeModule?: NativeModule | undefined)` can just be written as `constructor(nativeModule?: NativeModule)`

Reply:
I resolved the whitepace issues, but running `npm run lint react-native` from the README.md contribution section results in an error in a dependency project.

> definitely-typed@0.0.3 lint /Users/kendavidson/git/DefinitelyTyped
> dtslint types "react-native"

Error: Errors in typescript@4.2 for external dependencies:
../react/index.d.ts(38,22): error TS2307: Cannot find module 'csstype' or its corresponding type declarations.

Reply:
@alloy This csstypes thing is tripping off many new contributors. Can anything be don about it other than local install? Maybe add this as devDependancy?

Reply:
I'm glad it's not just me.